requirements.txt

```diff
@@ -1,3 +1,14 @@
-pyperclip==1.8.2
-python-dotenv==0.21.0
-selenium==4.11
+# This file was autogenerated by uv via the following command:
+#    uv pip compile pyproject.toml -o requirements.txt
+greenlet==3.0.3
```

Comment:
Just in case people haven't switched to poetry or uv and still using pip. 
Haven't checked to see if you can install dependencies from pyproject.toml with pip.